fuzzbucket/box.py

```diff
@@ -55,6 +56,10 @@ def from_ec2_dict(cls, instance: dict) -> "Box":
             public_ip=instance.get("PublicIpAddress", None),
         )
 
+        az = instance.get("Placement", {}).get("AvailabilityZone")
+        if az is not None:
+            box.region = str(az[:-1])
```

Comment:
Don't you know this by virtue of where FB is deployed? That is, you are talking to the AWS API endpoint for the region in order to get information about the instance.

Reply:
Yes! This is a bit convoluted, but this API call is already happening server side and I figured there wasn't much harm in using the data that was there rather than storing it somewhere and merging it into the response 🤷🏼